### ISSUE TITLE
help printed twice on Linux

### ISSUE THREAD

**Issue by jvolkening** [smith-chem-wisc/FlashLFQ#72]
A minor thing -- the help text is printed twice when run on Linux using Mono (this is with 0.1.108):
```
jvolkening@samson:~$ mono ~/scratch/FlashLFQ/CMD.exe --help
Valid arguments:
--idt [string|identification file path (TSV format)]
--rep [string|directory containing spectrum data files]
--out [string|output directory]
--ppm [double|ppm tolerance]
--iso [double|isotopic distribution tolerance in ppm]
--sil [bool|silent mode]
--int [bool|integrate features]
--mbr [bool|match between runs]
--mrt [double|maximum MBR window in minutes]
--chg [bool|use only precursor charge state]
--rmm [bool|require observed monoisotopic mass peak]
--nis [int|number of isotopes required to be observed]
--nor [bool|normalize intensity results]
--pro [bool|advanced protein quantification]

Valid arguments:
--idt [string|identification file path (TSV format)]
--rep [string|directory containing spectrum data files]
--out [string|output directory]
--ppm [double|ppm tolerance]
--iso [double|isotopic distribution tolerance in ppm]
--sil [bool|silent mode]
--int [bool|integrate features]
--mbr [bool|match between runs]
--mrt [double|maximum MBR window in minutes]
--chg [bool|use only precursor charge state]
--rmm [bool|require observed monoisotopic mass peak]
--nis [int|number of isotopes required to be observed]
--nor [bool|normalize intensity results]
--pro [bool|advanced protein quantification]
```

**Comment by rmillikin** [comment]
Thanks, I'll see if I can track that down.

I have heard that Mono has issues with RAM management (garbage collection). Have you experienced any problems with that?

I can make a .NET core version of the command-line of FlashLFQ so you can run that on Linux... I don't have easy access to a Linux machine so it will be difficult for me to troubleshoot. But I think that might be helpful for you and probably others.

**Comment by jvolkening** [comment]
> I have heard that Mono has issues with RAM management (garbage collection). Have you experienced any problems with that?

I haven't been keeping an eye on memory usage (running on a machine with 512 GB RAM) but I can pay closer attention in the future to see if it is relatively uniform or increases over time with large numbers of inputs. Generally, FlashLFQ has been working fine so far for me with Mono, apart from the issues I've opened here.

I know next to nothing about .NET programming - what's the difference between a .NET core build and what you are distributing now?

**Comment by acesnik** [comment]
It looks like the CMD project [targets .NET Framework](https://github.com/smith-chem-wisc/FlashLFQ/blob/master/CMD/CMD.csproj#L156). Rob, have you considered switching over to .NET Core? I haven't tried installing the .NET Core environment in Linux/Mac, but that should allow FlashLFQ to be run without Mono.

**Comment by acesnik** [comment]
Here are the instructions for [installing .NET Core on Linux](https://www.microsoft.com/net/download)

**Comment by rmillikin** [comment]
Fixed in version 1.0.1
